src/controllers/message.controller.js

```diff
@@ -0,0 +1,25 @@
+import { Message } from '../models/Message.model.js';
+import { EventEmitter } from 'events';
+
+export const emitter = new EventEmitter();
+
+export const getMessages = async (req, res) => {
+  const whereClauses = {};
+  const { roomId } = req.query;
+  whereClauses.roomId = roomId;
+  res.send(await Message.findAll({ where: whereClauses }));
```

Comment:
Consider adding error handling for the database operations. Currently, if the `Message.findAll` operation fails, your server might crash. You can add a try-catch block to handle any potential errors and send an appropriate response to the client.